src/context/AuthContext.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { createContext, useContext, useReducer, useState } from "react";
 import { AuthProvider } from "src/utils/Auth";
-import { VAR_USER_DETAILS } from "src/utils/Route";
+import { VAR_ENCODE_TOKEN, VAR_USER_DETAILS, VAR_USER_ID } from "src/utils/Route";
```

Comment:
You can also avoid the VAR prefix.

Reply:
it is only for to make them unique variable to store them in local